### DISCUSSION TITLE
How to input data

### DISCUSSION THREAD

**Discussion by MandeepCodes** [ananthakumaran/paisa#184] [General]
I mean its a great app to view money management. 
but either I dont understand how to input expenses, or is it really every time I have to go to editor and add expense.

**Comment by ananthakumaran** [comment]
Paisa supports file based [import](https://paisa.fyi/reference/import/). If you are looking for automatic bank sync, that is not supported.

**Comment by MandeepCodes** [comment]
@ananthakumaran No sir, I am looking for a button where I could enter a expense, transfer, income in ledger file. 
For example in https://github.com/tgrosinger/ledger-obsidian

**Reply by MandeepCodes** [reply]
Please respond. I really like your application. I just want to know how to use it.

**Reply by ananthakumaran** [reply]
There is no form based input. Only import or manual entry via editor is supported as of now

**Reply by MandeepCodes** [reply]
I see, is it on the roadmap ?

**Reply by ananthakumaran** [reply]
No, try the editor. Once you get used to it, it will be faster than form based entry
